lib/packages/core/lib/src/views/home/home_feed.dart

```diff
@@ -60,6 +60,7 @@ class _LMChatHomeScreenState extends State<LMChatHomeScreen> {
 
   @override
   void dispose() {
+    homeFeedPagingController.itemList?.clear();
```

Comment:
we can dispose the paging controller here, can we check if there is a dispose method available for paging controller.